src/main/java/org/springframework/data/elasticsearch/core/convert/MappingElasticsearchConverter.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2013-2019 the original author or authors.
+ * Copyright 2019 the original author or authors.
```

Comment:
Please retain copyright years. IntelliJ changes this typically when moving files around.

Reply:
oops, I had created a new file, and moved the parts from the old code over part by part, and renaming then, and lost this